modules/microshift-disabling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-driver_{context}"]
+= Disabling deployments that run the CSI driver implementations
+
+Before you disable the CSI driver implementation pods, you must manually stop any user workloads that have attached volumes. Use the following procedure to disable installation of the CSI implementation pods.
+
+[NOTE]
+====
+{microshift-short} will not delete CSI driver implementation pods. You must configure {microshift-short} to disable installation of the CSI driver implementation pods during the startup process.
+====
+
+.Procedure
+
+. Disable installation of the CSI driver by entering the `driver` value under the `storage` section of the {microshift-short} configuration file in `/etc/microshift/config.yaml`:
++
+[source,yaml]
+----
+# ...
+  storage
+   driver:
+   - "none" <1>
+# ...
+----
+<1> Valid values are `none` or `lvms`.
++
+[NOTE]
+====
+If the `driver` value is empty or null, {microshift-short} defaults to deploying LVMS.
```

Comment:
Maybe also repeat here that by default, the value is undefined and LVMS is deployed.

Reply:
done